deploy-manage/monitor/autoops.md

```diff
@@ -56,9 +56,7 @@ AutoOps currently monitors only {{es}}, not the entire {{stack}}. Any deployment
 In this section, you'll find the following information:
 
 * How to [open AutoOps](/deploy-manage/monitor/autoops/ec-autoops-how-to-access.md) for your deployment.
-* The contents of [AutoOps events](/deploy-manage/monitor/autoops/ec-autoops-events.md).
-* The [views](/deploy-manage/monitor/autoops/views.md) AutoOps offers to gain insight into facets of your deployment.
-* [Notification settings](/deploy-manage/monitor/autoops/ec-autoops-notifications-settings.md) that allow you to specify when and how to be notified.
-* [Event settings](/deploy-manage/monitor/autoops/ec-autoops-event-settings.md) that allow you to fine-tune when events are triggered, and a method to [dismiss](/deploy-manage/monitor/autoops/ec-autoops-dismiss-event.md) certain categories of events.
+* What [events](/deploy-manage/monitor/autoops/ec-autoops-events.md) are and how you can configure [event settings](/deploy-manage/monitor/autoops/ec-autoops-event-settings.md) and [notification settings](/deploy-manage/monitor/autoops/ec-autoops-notifications-settings.md) for events.
+* The [views](/deploy-manage/monitor/autoops/views.md) AutoOps offers to gain insight into your deployment.
```

Comment:
```suggestion
* Which [views](/deploy-manage/monitor/autoops/views.md) AutoOps offers to gain insight into your deployment.
```
I think "Which" fits better with how the previous two bullets start ("How", "What") than "The".